src/test/java/fr/techad/edc/popover/swing/example/Main.java

```diff
@@ -106,4 +112,22 @@ private static void createAndShowGUI() {
         f.pack();
         f.setVisible(true);
     }
+
+    private static JComboBox<String> createLangSelector() {
```

Comment:
Add this new option in the documentation.

Reply:
Done